package.json

```diff
@@ -11,23 +11,19 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"scripts": {
-		"build": "npm run policy-check && npm run layer-check && npm run build:genver && npm run build:compile && npm run lint && npm run build:docs",
-		"build:ci": "npm run build:genver && pnpm run -r --stream build:compile",
-		"build:compile": "pnpm run -r --stream build:compile",
-		"build:docs": "pnpm run -r --no-sort --stream build:docs",
-		"build:fast": "fluid-build --root .",
-		"build:full": "npm run build:genver && npm run build:full:compile && npm run lint && npm run build:docs",
-		"build:full:compile": "pnpm run -r --stream build:full:compile",
+		"build": "npm run policy-check && npm run layer-check && fluid-build --task build",
+		"build:compile": "fluid-build --task compile",
+		"build:docs": "fluid-build --task build:docs",
+		"build:fast": "fluid-build --task build",
+		"build:full": "fluid-build --task full",
+		"build:full:compile": "fluid-build --task compile --task webpack",
 		"build:gendocs": "copyfiles server/routerlicious/_api-extractor-temp/** ./_api-extractor-temp/doc-models/ -f -V && cd docs && npm run build",
-		"build:genver": "pnpm run -r --no-sort --stream build:genver",
 		"bump-version": "fluid-bump-version --root .",
 		"bundle-analysis:collect": "npm run webpack:profile && flub generate bundleStats",
 		"bundle-analysis:run": "flub run bundleStats --dangerfile build-tools/packages/build-cli/lib/lib/dangerfile.js",
 		"changeset": "flub changeset add",
 		"check:versions": "flub check buildVersion -g client --path .",
-		"ci:build": "npm run build:genver && pnpm run -r --stream build:compile",
-		"ci:build:docs": "pnpm run -r --no-sort --stream ci:build:docs",
-		"ci:eslint": "pnpm run -r --no-sort --stream eslint",
+		"ci:build": "fluid-build --task ci:build",
```

Comment:
Won't this try to run `ci:build` on each package (and kind-of-fail because not all packages have that script)? I would've guessed `fluid-build --task build:compile` but I'm not super familiar with fluid-build.

Similar concertn with `build:compile -> compile`, `build:full -> full`, and `build:full:compile -> compile`.

Reply:
I added capability to `fluid-build` to define `task` that are not `scripts`, i.e., it won't trigger scripts.  
The tasks are declared int `fluidBuild.config.cjs` at the root, with the possibility to override the dependencies in the per package's `package.json`